### PR TITLE
Improve the button position and ui of event dashboard

### LINKED ISSUE #2947: Events Dashboard: Move buttons to first column and improve design
In the events dashboard at https://eventyay.com/events/live please move the buttons in the column "Actions" to the front below the name in horizontal order. Increase the size of the buttons and improve logic. Show 4 buttons:
* Event Dashboard button
* View Event button -> to public event page
* Edit Event

Don't show the "delete event button" in the dashboard.

Screenshot from 2019-05-18 13-46-51

### PR DESCRIPTION
**Pull request by mohitm15**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
- It set the order of the buttons horizontally
- Increases the size of the buttons.
- Hides the `delete` button from event dashboard

#### Changes proposed in this pull request:

- Update `cell-buttons.hbs`

**Screenshot**
Screenshot 2019-05-18 23:37:22

Fixes #2947

### CODE REVIEW

**Comment by mohitm15** [comment]
@mariobehling please review

**Comment by mariobehling** [comment]
Screenshot from 2019-05-18 20-14-03

The buttons should move to the front area in horizontal order as follows:

Increase the size of the buttons and improve logic. Show 3 buttons:
* Event Dashboard button
* View Event button -> to public event page
* Edit Event

**Comment by mohitm15** [comment]
@mariobehling can you please clarify the purpose of first button

**Comment by mohitm15** [comment]
@mariobehling Now review please 
Screenshot 2019-05-19 00:02:50

File: app/templates/components/ui-table/cell/cell-buttons.hbs [hunk 1]
```diff
@@ -1,17 +1,20 @@
-<div class="ui vertical compact basic buttons">
-  {{#ui-popup tagName='a' click=(action moveToDetails record.identifier) content=(t 'View') class='ui icon button' position='left center'}}
+<div class="ui horizontal compact basic buttons">
```
Comment (uds5501) [inline app/templates/components/ui-table/cell/cell-buttons.hbs]: Do the following : 
- Take this code and paste it under the template which handles the Name column
- Delete this file.
- Make the button size responsive

**Comment by mariobehling** [comment]
> Screenshot from 2019-05-18 20-14-03
> 
> The buttons should move to the front area in horizontal order as follows:
> 
> Increase the size of the buttons and improve logic. Show 3 buttons:
> 
> * Event Dashboard button
> * View Event button -> to public event page
> * Edit Event

**Comment by kushthedude** [comment]
@mohitm Can you resolve it fast ?

**Comment by ritikamotwani** [comment]
@mohitm15 , any updates?

**Comment by uds5501** [comment]
@ritikamotwani shall I provide an alternative PR?

**Comment by ritikamotwani** [comment]
Closing due to inactivity. 
@uds5501 , sure.

**Comment by uds5501** [comment]
@ritikamotwani Okay, providing one within a few minutes!

**Pull request closed without merging**